docs/alerting/email-destination.md

```diff
@@ -0,0 +1,44 @@
+
+## Manage Senders
+Manage senders allows you to configure and manage **Sender** email addresses. Unless at least one sender is configured in the settings, the **Sender** dropdown list will be empty. 
+
+To configure a sender email, do the following.
+
+1. Choose **Manage senders**.
+1. In the **Manage email senders** modal window, choose **Add sender**. 
+1. Next, choose **New sender**. Multiple senders can be added one at a time.
+1. Enter a **Sender name**. Duplicate names are not allowed. You must enter a unique and descriptive name by using valid symbols, numbers, or words. A unique and descriptive name is also helpful for search queries.
+1. Enter the **Email address**, SMTP **Host** (e.g. `smtp.gmail.com` for a Gmail account), and the **Port** number.
+1. You can choose to use an **Encryption method** or leave it as **None**. SSL or TLS are standard protocols to secure your email. Enter your secure credentials to verify access to the account i.e. your email ID and password. You can enter these credentials in the Elasticsearch Keystore using the CLI. Run the following commands (in the bin directory of your Elasticsearch directory) to enter your username and password. The `<sender_name>` is the **Sender** name you entered.
+    `./bin/elasticsearch-keystore add opendistro.alerting.destination.email.<sender_name>.username` 
+    `./bin/elasticsearch-keystore add opendistro.alerting.destination.email.<sender_name>.password`
+1. Choose **Save** to save the configuration and create the sender.
+
+Once the sender is created, the sender account is available to be selected when creating an email destination, and can be used for multiple destinations. However, a destination only allows one sender. 
+
+## Manage email groups or recipients 
+You can use **Manage email groups** to create and manage a list of email addresses that you frequently send to different groups. Multiple recipients can be grouped into mailing lists based on the type of alert they should receive. For example, a certain type of alert may be triggered for the DevOps team (one email group) and another for the engineering team (another email group). 
+
+You can enter individual email addresses, or an email group in the **Recipients** field. For email groups, you can pre-create a group using **Manage email groups**.
+
+To create and manage email groups, do the following.
+1. Choose **Manage email groups**, and then **Add email group**.
+1. Choose **New email group**.
+1. Enter the **Email group name**. Duplicate names are not allowed. You must enter a unique and descriptive name by using valid symbols, numbers or words. 
```

Comment:
Seems excessive. You can just say "Enter a unique name for the email group."

Reply:
done